tests/test_lineshapes_models.py

```diff
@@ -270,3 +270,20 @@ def test_splitlorentzian_prefix():
     mod2 = models.SplitLorentzianModel(prefix='prefix_')
     par2 = mod2.make_params(amplitude=1.0, center=0.0, sigma=0.9, sigma_r=1.3)
     par2.update_constraints()
+
+
+def test_guess_from_peak():
+    x = np.linspace(-5, 5)
+    amplitude = 0.8
+    center = 1.7
+    sigma = 0.3
+    y = lineshapes.lorentzian(x, amplitude=amplitude, center=center, sigma=sigma)
+
+    model = models.LorentzianModel()
+    guess_increasing_x = model.guess(y, x=x)
+    guess_decreasing_x = model.guess(y[::-1], x=x[::-1])
+
+    for guess in [guess_increasing_x, guess_decreasing_x]:
```

Comment:
I don't think there is a need to loop over the result of the ```guess```, correct? If so, can you please change this to ```assert guess_increasing_x == guess_decreasing_x```. That passes with this PR, but doesn't when using `master`.

Reply:
I included ``assert guess_increasing_x == guess_decreasing_x.``
and then just looped over one of the guesses. Is that what you meant? The loop is checking consistency between the guess and the values used to create the test data.

Reply:
thank you! I would be lying if I said that's what I meant( because I misread the intention of the loop); but yes, now it's doing two useful things ;)